### DISCUSSION TITLE
(disregard) Announcement: End of microCOVID Project - 2023-03-10

### DISCUSSION THREAD

**Discussion by apiology** [microCOVID/microCOVID#1791] [General]
Johns Hopkins University has [announced the end of their live COVID-19 data reporting](https://github.com/govex/COVID-19/issues/600) effective March 10, 2023.

microCOVID relies heavily on that data and will not function past that date in its current form.

In light of this sad news, I hope you'll join me in thanking the [folks who made the microCOVID project happen](https://www.microcovid.org/paper/1-intro#authors), with a special focus on those that created this from nothing in a very scary time.  

They gave us an evidence-based beacon through a fog of uncertainty and we are in their debt.

**Comment by apiology** [comment]
If you're interested in what we can do to salvage the value of the site, I do have some ideas, and maybe you do, too.  The likely skills are frontend software engineering (TypeScript/React), data engineering (Python), and/or assessing scientific papers.  Any one of the above would be useful.  While I don't want to raise hopes given the effort involved, I created [a thread](https://github.com/microCOVID/microCOVID/discussions/1792) for details on what can be done.  I'll keep that thread limited to folks who in a position to help and I will moderate it strictly to keep it focused.  Don't worry, though, the post you're on is open to everyone's questions and comments.

**Comment by hypodium** [comment]
Sad news indeed. :-( We've come to rely heavily on microcovid.org, particularly as testing data becomes more sparse and/or stale, we override the location-based data and enter our own estimates. Would it be possible to continue that feature?

**Comment by MichelleGlauser** [comment]
Oh, no! What will I point to now to turn down social events? Thank you so, so much to everyone who made this resource possible!

**Comment by Jar3d8** [comment]
Is there any way to keep it running without up-to-date data? Like could you take an average of the covid cases that *have been* in an area over the past 3 years and treat that as the value used in that part of the calculation? Or just pick a number that assumes a fixed percentage of the population in that area has covid given how virtually all strategies for the mitigation of spread have been abandoned?

I only just discovered this tool and it has been extremely helpful for me, and I know it has been extremely helpful for other people too, and I really want it to continue existing.

**Comment by boxlike-supply** [comment]
What are the costs of hosting + registration for the site?  I contributed in the past and can send more again but I think there was some doubt about whether the [donate link](https://www.microcovid.org/contact) is still hooked up to someone actively involved in the project.

**Comment by apiology** [comment]
Hey all - I wanted to provide an update.

Complete and valid updates from JHU have clearly slowed down; we went almost a week without a successful prevalence run.  We're up to date as of yesterday, but it's quite possible that the effective end date is closer than 3/10 as a result.

I appreciate the ideas and leads!  To level-set, though, hours from skilled folks are the missing piece, not imagination on potential next steps or the comparatively cheap domain registration/tools.

Based on experience over the last few months, I do not realistically see that changing at the scale we need, but hope springs eternal.  If you have a significant amount of time to spend and skills in fund raising, modeling/analysis, frontend development, or data engineering, [contact me](https://calendly.com/apiology/microcovid) and I'd be happy to pair with you on working out a plan / learning the toolkit / whatever is needed.

**Comment by octern** [comment]
Just a note that this announcement is now out of date. We are switching the site to manual prevalence entry, which is inconvenient but hopefully will allow the site to remain useful!

**Comment by zerotrickpony** [comment]
Ya'll the site is looking really good now! Thank you so much for making these changes!!!
